features/api_token_info.feature

```diff
@@ -0,0 +1,125 @@
+      {"name": "esm-apps", "available": true}
+      {"name": "esm-infra", "available": true}
+      {"name": "fips", "available": true}
+      {"name": "fips-preview", "available": false}
+      {"name": "fips-updates", "available": true}
+      {"name": "landscape", "available": false}
+      {"name": "livepatch", "available": true}
+      {"name": "realtime-kernel", "available": false}
+      {"name": "ros", "available": true}
+      {"name": "ros-updates", "available": false}
+      {"name": "usg", "available": true}
+      """
+    And if `<release>` in `jammy` then output is:
+      """
+      {"name": "anbox-cloud", "available": true}
+      {"name": "cc-eal", "available": false}
+      {"name": "esm-apps", "available": true}
+      {"name": "esm-infra", "available": true}
+      {"name": "fips", "available": true}
+      {"name": "fips-preview", "available": false}
+      {"name": "fips-updates", "available": true}
+      {"name": "landscape", "available": false}
+      {"name": "livepatch", "available": true}
+      {"name": "realtime-kernel", "available": false}
+      {"name": "ros", "available": true}
+      {"name": "ros-updates", "available": false}
+      {"name": "usg", "available": true}
+      """
+    And if `<release>` in `noble` then output is:
+      """
+      {"name": "anbox-cloud", "available": true}
+      {"name": "cc-eal", "available": false}
+      {"name": "esm-apps", "available": true}
+      {"name": "esm-infra", "available": true}
+      {"name": "fips", "available": false}
+      {"name": "fips-preview", "available": false}
+      {"name": "fips-updates", "available": false}
+      {"name": "landscape", "available": true}
+      {"name": "livepatch", "available": true}
+      {"name": "realtime-kernel", "available": true}
+      {"name": "ros", "available": false}
+      {"name": "ros-updates", "available": false}
+      {"name": "usg", "available": false}
+      """
+    And if `release` in `oracular` then output is:
+      """
+      {"name": "anbox-cloud", "available": true}
+      {"name": "cc-eal", "available": false}
+      {"name": "esm-apps", "available": true}
+      {"name": "esm-infra", "available": true}
+      {"name": "fips", "available": true}
+      {"name": "fips-preview", "available": false}
+      {"name": "fips-updates", "available": true}
+      {"name": "landscape", "available": false}
+      {"name": "livepatch", "available": true}
+      {"name": "realtime-kernel", "available": false}
+      {"name": "ros", "available": true}
+      {"name": "ros-updates", "available": false}
+      {"name": "usg", "available": true}
+      """
+
+    Examples: ubuntu release
+      | release  | machine_type  |
+      | xenial   | lxd-container |
+      | bionic   | lxd-container |
+      | focal    | lxd-container |
+      | jammy    | lxd-container |
+      | noble    | lxd-container |
+      | oracular | lxd-container |
+
+  Scenario Outline: Error when no token is provided
```

Comment:
I don't think we need a different container for this, this can be the first check on the previous scenario